eva/optimizer/rules/rules.py

```diff
@@ -692,16 +695,38 @@ def apply(self, before: LogicalGet, context: OptimizerContext):
         )
         if config_batch_mem_size:
             batch_mem_size = config_batch_mem_size
-        after = SeqScanPlan(None, before.target_list, before.alias)
-        after.append_child(
-            StoragePlan(
-                before.dataset_metadata,
-                batch_mem_size=batch_mem_size,
-                predicate=before.predicate,
-                sampling_rate=before.sampling_rate,
+        ray_enabled = ConfigurationManager().get_value("experimental", "ray")
```

Comment:
@kaushikravichandran Can we consider creating a new file under the experimental directory and putting in it the newer rules/modification?  Then we can simply add these rules to the ruleset based on the `ray_enabled` flag.

Reply:
My motivation is to segregate the `ray` related changes as much as possible.